tests/integration_tests_control_files/advection_explicit_periodic/2D_advection_explicit_periodic_energy_weight_adjusted.prm

```diff
@@ -7,7 +7,7 @@ set dimension = 2
 
 set use_weak_form = false
 
-set overintegration = 0
+set overintegration = 2
```

Comment:
why is OI added?

Reply:
To check the overintegrated case. Before this PR the test would fail for overintegrated, now it passes.